articles/cosmos-db/mongodb-custom-commands.md

```diff
@@ -215,6 +216,7 @@ The following table describes the parameters within the command:
 | `offerThroughput` | `int` | Optional | Provisioned throughput to set on the database. If this parameter is not provided, it will default to the minimum, 400 RU/s. * To specify throughput beyond 10,000 RU/s, the `shardKey` parameter is required.|
 | `shardKey` | `string` | Required for collections with large throughput | The path to the Shard Key for the sharded collection. This parameter is required if you set more than 10,000 RU/s in `offerThroughput`.  If it is specified, all documents inserted will require this key and value. |
 | `autoScaleSettings` | `Object` | Required for [Autoscale mode](provision-throughput-autoscale.md) | This object contains the settings associated with the Autoscale capacity mode. You can set up the `maxThroughput` value, which describes the highest amount of Request Units that the collection will be increased to dynamically. |
+| `indexes` | `Array` | Optional indexes (3.6+ accounts only). When present, an index on _id is required. Each entry in the array must include a key of one or more fields, and may contain index options. For example this creates a compound unique index on the fields a and b: ````{key: {a: 1, b: 1},unique: true}````.
```

Comment:
```suggestion
| `indexes` | `Array` | Optionally configure indexes (supported for 3.6+ accounts only). When present, an index on _id is required. Each entry in the array must include a key of one or more fields, and may contain index options. For example, to create a compound unique index on the fields a and b use this entry: `{key: {a: 1, b: 1},unique: true}`.
```

Reply:
Also recommend splitting the content after first sentence into the description column (so add a `|` after "... only)." and before "When present..."

Sneha's comment would apply to the other example as well, (similar structure), but it doesn't have separate required/description column, so my comment wouldn't apply without a bigger reformatting.